_ko/tour/tour-of-scala.md

```diff
@@ -2,51 +2,66 @@
 layout: tour
 title: 들어가며
 
-discourse: false
+discourse: true
 
 partof: scala-tour
 
 num: 1
-language: ko
 
 next-page: basics
+
+redirect_from: "/tutorials/tour/tour-of-scala.html"
+
+next-page: basics
+
+language: ko
 ---
 
-스칼라는 정확하고 명쾌하며 타입 세이프한 방식으로 일반적인 유형의 프로그래밍 패턴을 표현하기 위해 설계된 새로운 다중 패러다임 프로그래밍 언어다. 스칼라는 객체지향과 함수형 언어를 자연스럽게 통합해준다.
+## 투어를 환영합니다
+이 투어에서는 스칼라에서 가장 자주 사용되는 기능을 요약하여 소개하며, 스칼라 초보자를 그 대상으로 합니다.
+
+완전한 언어 학습서가 아닌 간단한 둘러보기만 제공하므로, 필요하면 [책](/books.html)을 구해보거나 [다른 자료](/learn.html)를 검토하세요.
```

Comment:
It would be better to make the reader more comfortable reading through paraphrase. 

**완전한 언어 학습서**가 아닌 **간단한 둘러보기**만 **제공하므로**, 필요하면 책을 구해보거나 다른 자료를 검토하세요.
**언어 전체를 다루는 튜토리얼**이 아닌 **간단히 둘러보기**입니다. 자세히 다루고 싶다면, 책을 구하거나 다른 자료를 찾아보세요.